src/charselect.cpp

```diff
@@ -190,6 +190,50 @@ void Courtroom::char_clicked(int n_char)
   }
 }
 
+void Courtroom::on_char_button_context_menu_requested(const QPoint &pos)
+{
+  AOCharButton* button = qobject_cast<AOCharButton*>(sender());
+  int n_char = ui_char_button_list.indexOf(button);
+  if (n_char == -1)
+  {
+    return;
+  }
+
+  QString char_name = char_list.at(n_char).name;
+  QString char_ini_path = ao_app->get_real_path(
+        ao_app->get_character_path(char_name, "char.ini"));
+
+  if (!file_exists(char_ini_path)) {
+    call_error(tr("Could not find character (char.ini) for %1").arg(char_name));
+    return;
+  }
+
+  QMenu *menu = new QMenu(this);
+  menu->addAction(QString("Edit " + char_name + "/char.ini"), this,
+                  [=] { QDesktopServices::openUrl(QUrl::fromLocalFile(char_ini_path)); }
+  );
+  menu->addSeparator();
+  menu->addAction(QString("Open character folder " + char_name), this,
+                  [=] {
+    QString p_path = ao_app->get_real_path(VPath("characters/" + char_name + "/"));
+    if (!dir_exists(p_path)) {
+      return;
+}
+    QDesktopServices::openUrl(QUrl::fromLocalFile(p_path));
+  }
+  );
+  menu->addAction(QString("Open characters folder"), this,
+                  [=] {
+    QString p_path = ao_app->get_real_path(VPath("characters/"));
+    if (!dir_exists(p_path)) {
+      return;
+}
+    QDesktopServices::openUrl(QUrl::fromLocalFile(p_path));
```

Comment:
Not a useful action. I would expect there to be many character folders. Which one would be selected?